StatisticsTests/GraphicalRelationships/GraphicalTests.cs

```diff
@@ -12,36 +13,40 @@ public class GraphicalTests
     {
         /// <summary>
         /// Test data based on Table 2.13 from "Uncertainty Estimates for Grahpical (Non-Analytic) Frequency Curves - HEC-FDA Technical Reference" CPD-72a
+        /// Standard deviations computed externally 
         /// </summary>
         static double[] exceedanceProbabilities = new double[] { .99, .95, .90, .85, .8, .75, .7, .65, .6, .55, .5, .45, .4, .35, .3, .25, .2, .15, .1, .05, .02, .01, .005, .0025 };
         static double[] quantileValues = new double[] { 6.6, 7.4, 8.55, 9.95, 11.5, 12.7, 13.85, 14.7, 15.8, 16.7, 17.5, 18.25, 19, 19.7, 20.3, 21.1, 21.95, 23, 24.2, 25.7, 27.4, 28.4, 29.1, 29.4 };
-        static double[] standardDeviations = new double[] { 0.482052902, 1.055902721, 1.710592003, 2.355386115, 2.459674775, 2.275377716, 2.049390153, 2.079746078, 2.19089023, 1.891130614, 1.732952683, 1.66864466, 1.588395417, 1.386497386, 1.434573107, 1.59760563, 1.699411663, 1.796480935, 1.811215062, 1.949358869, 2.113084239, 2.521507486, 2.521507486, 2.521507486 };
+        static double[] standardDeviations = new double[] { 0.366239124, 0.366239124, 1.055902721, 1.710592003, 2.459674775, 2.275377716, 2.049390153, 2.079746078, 2.19089023, 2.19089023, 1.732952683, 1.66864466, 1.588395417, 1.386497386, 1.434573107, 1.59760563, 1.699411663, 1.796480935, 1.811215062, 1.949358869, 2.113084239, 2.113084239, 2.521507486, 2.102908039 };
         static int equivalentRecordLength = 20;
 
-        //THIS TEST SHOULD YIELD A NON-MONOTONIC FUNCTION
+
         [Fact]
         public void GraphicalFunction_Test()
         {
             
-            Graphical graphical = new Graphical(exceedanceProbabilities, quantileValues, equivalentRecordLength,.99,.01);
+            Graphical graphical = new Graphical(exceedanceProbabilities, quantileValues, equivalentRecordLength,.999,.001);
             graphical.ComputeGraphicalConfidenceLimits();
             double[] computedStandardDeviations = new double[standardDeviations.Length];
             double[] confirmExceedanceProbabilities = new double[standardDeviations.Length];
+            List<double> exceedanceProbabilityList = graphical.ExceedanceProbabilities.ToList();
             for (int i = 0; i < exceedanceProbabilities.Length; i ++)
             {
-                int idx = Array.BinarySearch(graphical.ExceedanceProbabilities, exceedanceProbabilities[i]);
-                if (idx > 0)
+                int idx = exceedanceProbabilityList.IndexOf(exceedanceProbabilities[i]); 
```

Comment:
The binary search didn't work because I was using exceedance probabilities (going from big to small). I suppose I could have moved to non-exceedance. 

I am not sure I understand how the two are that much different. Would you mind telling me a little more?

Reply:
Microsoft says IndexOf returns "the zero-based index of the first occurrence of item within the range of elements in the List<T> that extends from index to the last element, if found; otherwise, -1." which is exactly what I want.